### ISSUE TITLE
Nameplate doesn't update, because Neovim API args missmatch

### ISSUE THREAD

**Issue by hoschi** [veonim/veonim#653]
With these steps:
* git clone repo
* `npm ci` to install packages
* `npm start`

I get this:
![2019-10-29-203635_1528x1395_scrot](https://user-images.githubusercontent.com/163128/67802777-1a974f00-fa8c-11e9-865a-68e40f1459a6.png)

Where it doesn't update nameplates, because it can't find the window by id. This patch does fix it:
![2019-10-29-204010_601x404_scrot](https://user-images.githubusercontent.com/163128/67802902-4f0b0b00-fa8c-11e9-92b7-3b63e14f2783.png)

I thought my system `nvim` is out of date, but updated it and still have the error. Does `npm start` use a precompiled neovim?

My system:
Linux (Arch)
```
veonim-mine > node --version 
v10.14.1
veonim-mine > npm --version
6.9.0
veonim-mine > nvim --version
NVIM v0.5.0-132-g83b6d9f19
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -O2 -g -DMIN_LOG_LEVEL=3 -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/home/hoschi/.cache/yay/neovim-git/src/neovim-git/build/config -I/home/hoschi/.cache/yay/neovim-git/src/neovim-git/src -I/usr/include -I/home/hoschi/.cache/yay/neovim-git/src/neovim-git/build/src/nvim/auto -I/home/hoschi/.cache/yay/neovim-git/src/neovim-git/build/include
```

**Comment by breja** [comment]
Yeah veonim uses it's own bundled version of `nvim` defined in package.json
https://github.com/veonim/veonim/blob/e4644f2be2109d17ad57b406f254dd0ec2f89007/package.json#L71-L78

The latest versions of nvim changed the API of `win_pos` (as you pointed out).

I took your fix and applied it here: ea2e4c0242a952e8051806f08f1386d839841126

Thanks for the investigation and fix!

**Comment by hoschi** [comment]
Thanks!
